src/app/features/vendor/vendor-routing.module.ts

```diff
@@ -2,10 +2,12 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { VendorListingComponent } from './vendor-listing/vendor-listing.component';
 import { VendorCreationComponent } from './vendor-creation/vendor-creation.component';
+import { VendorViewComponent } from './vendor-view/vendor-view.component';
 
 const routes: Routes = [
   { path: '', component: VendorListingComponent },
   { path: 'creation', component: VendorCreationComponent },
+  { path: 'view', component: VendorViewComponent },
```

Comment:
Add vendor id in path to specify the selected vendor.

Reply:
Done that